packages/dataviews/src/view-grid.js

```diff
@@ -155,16 +194,20 @@ export default function ViewGrid( {
 		<>
 			{ hasData && (
 				<Grid
+					ref={ gridRef }
 					gap={ 6 }
 					columns={ 2 }
 					alignment="top"
 					className="dataviews-view-grid"
 					aria-busy={ isLoading }
+					onMouseEnter={ pointerIsWithinBoundsHandler }
```

Comment:
Curious why you added these events in Grid and not in GridItem..

Reply:
Mostly just to reduce the number of events being processed; we're manually adding and removing `keyup` and `keydown` event handlers to the body, and doing that every time the mouse pointer moves in and out of a grid item could get expensive.

The `key*` event handlers have to be on the body, because focus could be anywhere; putting them on each grid item would require them to have focus to function properly.